### PR TITLE
r_str_glob

### PR DESCRIPTION
**Pull request by thelightnet**

### CODE REVIEW

**Comment by Maijin** [comment]
Nop, this breaks Appveyor build -> https://ci.appveyor.com/project/radare/radare2-shvdd/build/0.10.3.9551#L296

**Comment by Maijin** [comment]
And use r2-ident tool for indentation and read https://github.com/radare/radare2/blob/master/CONTRIBUTING.md

File: libr/util/str.c [hunk 1]
```diff
@@ -1580,12 +1580,27 @@ R_API bool r_str_glob (const char* str, const char *glob) {
 	const char* cp = NULL, *mp = NULL;
 	if (!glob || !strcmp (glob, "*")) {
 		return true;
-	}
 	if (!strchr (glob, '*')) {
-		return strstr (str, glob) != NULL;
-	}
+    if (*glob == '^'){
```
Comment (radare) [inline libr/util/str.c]: use tabs

File: libr/util/str.c [hunk 2]
```diff
@@ -1580,12 +1580,27 @@ R_API bool r_str_glob (const char* str, const char *glob) {
...
+        if (*glob != *str) {
+          return false;
+        }
+        if (!*++glob) {
+          return true;
+        }
+        str++;
+      } 
+    } else {
+		  return strstr (str, glob) != NULL;
```
Comment (radare) [inline libr/util/str.c]: wrong indent

File: libr/util/str.c [hunk 3]
```diff
@@ -1580,12 +1580,27 @@ R_API bool r_str_glob (const char* str, const char *glob) {
...
+    }
+	}
+  if (*glob == '^'){
```
Comment (radare) [inline libr/util/str.c]: missing space after )

**Pull request merged**